src/main/java/de/blackforestsolutions/datamodel/ApiTokenAndUrlInformation.java

```diff
@@ -27,7 +27,11 @@
     private final String xOriginationIpKey;
     private final String xOriginationIp;
     private final String authorizationKey;
-    private final String authorization;
+    /**
+     * We need to update our bearer token for lufthansa request
```

Comment:
We allready had a discussion about that, maybe you explain in short terms why you choice was this way instead of the other ways we discussed.

Reply:
also xOriginationIp can be deleted bc it is not needed anymore